### PR TITLE
[DOCFIX] [branch-1.6] Update the class not found issue when using  SparkSQL

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Running-Spark-on-Alluxio.md [hunk 1]
```diff
@@ -200,17 +200,18 @@ cluster with `--num-executors=1`, Spark might place the only executor on `host2`
...
 
-## `Failed to login` Issues with Spark Shell
+## `Class alluxio.hadoop.FileSystem not found` Issues with SparkSQL
 
 To run the `spark-shell` with the Alluxio client, the Alluxio client jar will have to be added to the classpath of the
-Spark driver and Spark executors, as described earlier. However, sometimes Alluxio will fail to determine the security
-user and will result in an error message similar to: `Failed to login: No Alluxio User is found.` Here are some
-solutions.
+Spark driver and Spark executors, as [described earlier](Running-Spark-on-Alluxio.html#general-setup). 
+However, sometimes SparkSQL fail to save tables to Hive metastore when the table locations in in Alluxio with an error
```
Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: how about something like:

"... SparkSQL may fail to save tables to the Hive metastore (location in Alluxio), with an error ..."

File: docs/en/Running-Spark-on-Alluxio.md [hunk 2]
```diff
@@ -200,17 +200,18 @@ cluster with `--num-executors=1`, Spark might place the only executor on `host2`
 However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
 enough to prioritize placing the job on `host1`.
 
-## `Failed to login` Issues with Spark Shell
+## `Class alluxio.hadoop.FileSystem not found` Issues with SparkSQL
```
Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: SparkSQL and Hive Metastore

Comment (apc999) [reply docs/en/Running-Spark-on-Alluxio.md]: done

File: docs/en/Running-Spark-on-Alluxio.md (hunk 1 above)
Comment (apc999) [reply docs/en/Running-Spark-on-Alluxio.md]: fixed

**Pull request merged**